pontoon/base/formats/lang.py

```diff
@@ -183,8 +189,8 @@ def parse(path, source_path=None):
 
     try:
         children = LangVisitor().parse(content)
-    except ParsimoniousParseError as err:
+    except (ParsimoniousParseError, VisitationError) as err:
         wrapped = ParseError(u'Failed to parse {path}: {err}'.format(path=path, err=err))
-        raise wrapped, None, sys.exc_info()[2]
+        raise wrapped, None, sys.exc_info()[2]  # NOQA
```

Comment:
[flake8](https://flake8.readthedocs.org/en/latest/) was complaining about this syntax for raising an exception being deprecated in Python 3.2. `# NOQA` is a marker you can put on a line to tell flake8 to stop linting that specific line, and sometimes I sprinkle them around when I'm knowingly breaking a rule that I care about.

Interestingly, adding this comment didn't stop flake8 from complaining about this particular line, but I think that's a bug with my version of flake8, so I left it in.